.github/labeler.yml

```diff
@@ -1,3 +1,4 @@
+# NOTE: bas and darwin labels are handled by ofborg
```

Comment:
```suggestion
# NOTE: bsd and darwin labels are handled by ofborg
```
ofborg also sets the "cross" label
https://github.com/NixOS/ofborg/blob/5a4e743f192fb151915fcbe8789922fa401ecf48/ofborg/src/tasks/eval/nixpkgs.rs#L30-L35